doc/api/n-api.md

```diff
@@ -3148,7 +3148,7 @@ Afterward, additional manipulation of the wrapper's prototype chain may cause
 
 *Note*: Calling `napi_wrap()` a second time on an object that already has a
 native instance associated with it by virtue of a previous call to
-`napi_wrap()` will cause an error to be returned. If you wish to associate
+`napi_wrap()` will cause an error to be returned. In order to associate
```

Comment:
Nit: Get rid of `In order`:

> To associate another native instance with the given object...

Nit: Consider getting rid of `*Note*:` at the start of this paragraph.

Nit: Get rid of ``` that already has a native instance associated with it by virtue of a previous call to `napi_wrap()` ```. The sentence already says `a second time` and adding all that wordy context just makes it confusing. Also s/cause an error to be returned/return an error/. Also s/given//.  Also.... Aw heck, here's what I think this whole too-long paragraph should be reduced to:

> Calling `napi_wrap()` a second time on an object will return an error. To associate
> another native instance with the object, use `napi_remove_wrap()` first.